src/sql.rs

```diff
@@ -1479,6 +1479,24 @@ CREATE INDEX devmsglabels_index1 ON devmsglabels (label);
             }
             if exists_before_update {
                 disable_server_delete = true;
+
+                // Don't disable server delete if it was on by default (Nauta):
+                if let Some(provider) = get_provider_by_id(
+                    &context
+                        .get_config(Config::ConfiguredProvider)
+                        .await
+                        .unwrap_or_default(),
+                ) {
+                    if let Some(defaults) = &provider.config_defaults {
+                        if let Some(config) =
+                            defaults.iter().find(|d| d.key == Config::DeleteServerAfter)
+                        {
+                            if config.value == "1" {
```

Comment:
This check is not necessary and would even be wrong if we had some other provider with a different setting, like 1 month.